src/yunohost/log.py

```diff
@@ -106,7 +106,7 @@ def log_list(category=[], limit=None, with_details=False):
                     try:
                         metadata = yaml.safe_load(md_file)
                     except yaml.YAMLError:
-                        logger.warning(m18n.n('log_corrupted_md_file', file=md_path))
+                        logger.warning(m18n.n('log_corrupted_md_file', md_file=md_path))
```

Comment:
I will happily merge this and yolorelease it so we can investigate the bug, but can we also get the actual error message so we can know what's going on with those file ? (Since apparently the issue tends to disappear by itself as we saw with messages on the forum)

In fact, there's a moulinette helper called `read_yaml` that does exactly this ...

Reply:
Done